airbyte-config/models/src/test/java/io/airbyte/config/AirbyteProtocolConvertersTest.java

```diff
@@ -153,4 +155,9 @@ void testAnyOfAsObject() {
     assertEquals(schema, AirbyteProtocolConverters.toSchema(message.getCatalog()));
   }
 
+  @Test
+  void testEnumConversion() {
+    assertTrue(Enums.isCompatible(io.airbyte.protocol.models.SyncMode.class, io.airbyte.config.StandardSync.SyncMode.class));
```

Comment:
Wondering if we could use the `useExsitingJavaType` so we can reuse the protocol definitions in the config instead of doing the conversion (we don't have to do it now but what about for the future?)